build/fix-api-docs.js

```diff
@@ -0,0 +1,20 @@
+var replace = require("replace");
+var path = require('path')
+var apiPath = path.join(__dirname, '..', 'docs', 'api');
+
+var replacements = [
+  {find: /\/docs\/guides\/(.+)\.md/, replace: 'tutorial-$1.html'},
```

Comment:
All of these need the global flag to apply properly to all occurrences in the file.